server/controllers/MeController.js

```diff
@@ -33,6 +33,39 @@ class MeController {
     res.json(payload)
   }
 
+  // GET: api/me/item/listening-sessions/:libraryItemId/:episodeId
+  async getListeningSessions(req, res) {
+    const libraryItem = await Database.libraryItemModel.getOldById(req.params.libraryItemId)
+    const episode = (req.params.episodeId && libraryItem && libraryItem.isPodcast) ? libraryItem.media.getEpisode(req.params.episodeId) : null
```

Comment:
We can query using the new data model for this which will be much faster. There is a `podcastEpisode` model that can be queried directly